apps/docs/docs/schematics/usage.md

```diff
@@ -5,30 +5,52 @@ sidebar_label: Usage
 
 :::info
 
-We'll show how to use the `nest-commander-schematics` with the `@nestjs/cli`, but it works with `@angular/cli` and `nx` as well, as all of them use Angular's schematics under the hood.
+We'll show how to use the `nest-commander-schematics` with [`@nestjs/cli`](https://www.npmjs.com/package/@nestjs/cli), but it works with `@angular/cli` and `nx` as well, as all of them use Angular's schematics under the hood.
 
 :::
 
 ## Generating Commands
 
-To generate a simple command you can use
+To generate a simple command you can use the `command` schematic:
 
-```shell
+```
 nest g -c nest-commander-schematics command
 ```
 
-, from there the wizard will ask what the name of the command is and if you would like to add questions. You can choose no or provide a question set name at this point. You can also use `--default=true` to automatically set the `isDefault` option. `spec`, `flat`, `path`, and `sourceRoot` are also available options.
+from there the wizard will ask what the name of the command is and if you would like to add questions. You can choose _no_ or provide a question set name at this point.
+
+The available options for this command are the following:
+
+```
+--name=<name>         Command name.
+--path=<name>         The path to create the service.
+--sourceRoot=<name>   NestJS service source root directory.
+--flat                Whether or not a directory is created. (default: fale)
+--spec                Whether or not a spec file is generated. (default: true)
+--default             Whether or not the command is the default command for the CLI. (default: false)
+--question=<name>     The name of the related question set.
+```
 
 ### Generating Commands with Questions
 
 As mentioned above, you can use the `command` schematic and provide a question set name. You can also use `--question=<name>` to provide a name for the question set without waiting to answer the prompt.
```

Comment:
do notice that we can't pass any option to `command` schematics using `@nestjs/cli`

```
$ nest g -c nest-commander-schematics command --question=name
error: unknown option '--question'

$ nest g -c nest-commander-schematics --question=name command
error: unknown option '--question'
```

Reply:
Oh really? I swear I had that tested. Maybe it was with the ng CLI instead. I'll need to see what can be done about this

Reply:
do you think Nest CLI should support both usages (options along with `-c`, and the usual one when using custom collection)?

Reply:
looks like the `--name` option is being populated with the first arg

```
$  nest g command foo
? What is the name of the related question set? (Leave blank for none) 
Failed to execute command: node nest-commander-schematics:command --name=foo --no-dry-run --language="ts" --sourceRoot="src" --no-spec
```

Reply:
I think I'll need to look into fixing this in nestjs/nest-cli. Thanks for bringing it up